src/iroh.rs

```diff
@@ -380,6 +411,180 @@ impl VeilidIrohBlobs {
 
         return Ok(read);
     }
+    pub async fn create_collection(&self, collection_name: String) -> anyhow::Result<Hash> {
```

Comment:
```suggestion
    pub async fn create_collection(&self, collection_name: &String) -> anyhow::Result<Hash> {
```

This and other params should probably be borrows since we don't mutate them or pass them off anywhere else

Reply:
b242bac685d08ad7d41afe3a2d737029b4341674